test/test_jit.py

```diff
@@ -710,6 +710,44 @@ def check(x, y):
 
         self.assertTrue(check(x, y))
 
+    def test_matrix_transpose(self):
+        @torch.jit.script
+        def check(x):
+            return torch.equal(x.mT, x.transpose(-2, -1))
+
+        x = torch.rand(3, 4)
+        self.assertTrue(check(x))
```

Comment:
I think we should use `checkScript` instead of `torch.jit.script`. It verifies that a given script generates the same output as the Python version using the given inputs. So we don't need any additional equality check since we already have correctness checks at other places

```
def test_T_mT_H_mH(self):
    def T(x):
        return x.mT
    def mT(x):
        return x.mT
    def H(x):
        return x.H
    def mH(x):
        return x.mH
    
    x = torch.rand(3, 4)
    y = make_tensor((3, 4), device="cpu", dtype=torch.complex64)
    
    self.checkScript(T, (x, ))
    self.checkScript(mT, (x, ))
    self.checkScript(H, (x, ))
    self.checkScript(mH, (x, ))
    self.checkScript(T, (y, ))
    self.checkScript(mT, (y, ))
    self.checkScript(H, (y, ))
    self.checkScript(mH, (y, ))
```

Reply:
Added.

Reply:
I don't see it in the PR. did you push your local changes yet?